sched/sched/sched_timerexpiration.c

```diff
@@ -233,14 +231,23 @@ static uint32_t nxsched_cpu_scheduler(int cpu, uint32_t ticks,
 
   /* Returning zero means that there is no interesting event to be timed */
 
-#ifdef KEEP_ALIVE_HACK
```

Comment:
But we should remove the hack branch instead?

Reply:
I remove that patch